app/helpers/donors_helper.rb

```diff
@@ -0,0 +1,5 @@
+module DonorsHelper
+  def is_or_isnt_on_site(donor)
+    donor.location.grown_on_site ? "is" : "is not"
```

Comment:
`"is"` and `"is not"` isn't going to work in other languages (in the future), and sort of defeats the purpose of internationalization.

What do you think about having two separate I18n values -- one for the true case and one for the false case?